Revit_Core_Engine/Create/View/Viewport.cs

```diff
@@ -55,10 +58,11 @@ public static Viewport Viewport(this ViewSheet sheet, View view, ElementId viewp
             viewPort.ChangeTypeId(viewportTypeId);
             viewPort.Rotation = viewportRotation;
 
-            document.Regenerate();
-
             if (viewportPlacementPoint != null)
+            {
```

Comment:
`viewportPlacementPoint` will never be null now

Reply:
Right, I removed this check 👍